front/src/modules/simulationResult/SimulationResultExport/SimulationReportSheetScenario.tsx

```diff
@@ -0,0 +1,328 @@
+          <View style={styles.convoyAndRoute.convoy}>
+            <Text style={styles.convoyAndRoute.convoyTitle}> {t('convoy')}</Text>
+            <View style={styles.convoyAndRoute.convoyInfo}>
+              <View style={styles.convoyAndRoute.convoyInfoBox1}>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('speedLimitByTag')}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoData}>{speedLimitByTag || '-'}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('towedMaterial')}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoData}>-</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('maxSpeed')}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoData}>
+                  {`${Math.floor(rollingStock.max_speed * 3.6)} km/h`}
+                </Text>
+              </View>
+              <View style={styles.convoyAndRoute.convoyInfoBox2}>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('maxWeight')}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoData}>
+                  {`${Math.floor(rollingStock.mass / 1000)} t`}
+                </Text>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('referenceEngine')}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoData}>
+                  {rollingStock.metadata?.reference || '-'}
+                </Text>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('maxLength')}</Text>
+                <Text
+                  style={styles.convoyAndRoute.convoyInfoData}
+                >{`${rollingStock.length} m`}</Text>
+              </View>
+            </View>
+          </View>
+          <View style={styles.convoyAndRoute.route}>
+            <Text style={styles.convoyAndRoute.routeTitle}>{t('requestedRoute')}</Text>
+            <View style={styles.convoyAndRoute.stopTableContainer}>
+              <Table style={styles.convoyAndRoute.stopTable}>
+                <TH style={styles.convoyAndRoute.stopTableTH}>
+                  <View style={styles.convoyAndRoute.stopTableIndexWidth}>
+                    <TD aria-label="line-count" />
+                  </View>
+                  <View style={styles.convoyAndRoute.stopTableOpWidth}>
+                    <TD>{t('operationalPoint')}</TD>
+                  </View>
+                  <View style={styles.convoyAndRoute.stopTableChWidth}>
+                    <TD>{t('code')}</TD>
+                  </View>
+                  <View style={styles.convoyAndRoute.stopTableEndWidth}>
+                    <TD>{t('endStop')}</TD>
+                  </View>
+                  <View style={styles.convoyAndRoute.stopTableStartWidth}>
+                    <TD>{t('startStop')}</TD>
+                  </View>
+                </TH>
+                {operationalPointsList.map((step, index) => {
```

Comment:
results are shifted slightly to the left:
<img width="970" alt="Capture d’écran 2024-10-21 à 17 08 11" src="https://github.com/user-attachments/assets/77d9b132-f5f5-4cd5-bc88-3aabc8882252">

Reply:
Done